server/app/Http/Requests/UsersRequest/UserCreateRequest.php

```diff
@@ -8,16 +8,14 @@ class UserCreateRequest extends UserLoginRequest
 {
     private function SetNameRequest()
     {
-        $name = array([
-            'name' => ['bail', 'required', 'string', 'max:50'],
-        ]);
+        $name = array('bail', 'required', 'string', 'max:50');
 
-        array_push($this->reglas, $name);
+        $this->reglas['name'] = $name;
```

Comment:
No es necesario crear la variable `$name`, se puede asignar el arreglo directamente a `$this->reglas['name']`

```suggestion
        $this->reglas['name'] = ['bail', 'required', 'string', 'max:50'];
```

Reply:
Solucionado en 78c15191cedf9cb693845c39b7103b481a27a563 ✔